docs/source/tutorial/typesfuns.rst

```diff
@@ -1322,22 +1320,22 @@ for building lists. The general form is:
     [ expression | qualifiers ]
 
 This generates the list of values produced by evaluating the
-``expression``, according to the conditions given by the comma
-separated ``qualifiers``. For example, we can build a list of
-Pythagorean triples as follows:
+``expression``, according to the conditions given by the
+comma-separated ``qualifiers``. For example, we can build a list
+of Pythagorean triples as follows:
 
 .. code-block:: idris
 
     pythag : Int -> List (Int, Int, Int)
     pythag n = [ (x, y, z) | z <- [1..n], y <- [1..z], x <- [1..y],
                              x*x + y*y == z*z ]
 
-The ``[a..b]`` notation is another shorthand which builds a list of
-numbers between ``a`` and ``b``. Alternatively ``[a,b..c]`` builds a
+The ``[a..b]``-notation is another shorthand which builds a list of
```

Comment:
Having the form `x`-notation where `x` is an example of the notation in use, may not be conducive. We should either use the name of the notation (i.e. range or enumerate) or rewrite as: "The notation, `[a..b]`,...".

Reply:
@jfdm @CodingCellist Should I go ahead with this change?

Reply:
The errors refer to it as "list range" notation, but I don't if it can be used for things other than `List`?